Sources/PackageGraph/PackageGraphRoot.swift

```diff
@@ -47,14 +47,30 @@ public struct PackageGraphRoot {
     }
 
     /// The top level dependencies.
-    public let dependencies: [PackageDependency]
+    private let _dependencies: [PackageDependency]
+
+    public var dependencies: [PackageDependency] {
```

Comment:
The doc comment that's currently on `_dependencies` should either be moved here or duplicated, since you're much more likely to look up docs for a public property than a private one.

Reply:
That makes sense to me.